internal/graph/present/present.go

```diff
@@ -131,7 +131,7 @@ func convert(err error) error {
 			Reason: err.Error(),
 			Code:   ErrorRetryable,
 		}
-	case errors.As(err, &dead), errors.As(err, &ref):
+	case errors.Is(err, dead), errors.As(err, &ref):
```

Comment:
Ah man, I struggle with the UX of `errors.Is` and `errors.As`. Whoops. 😢

Reply:
at first I thought this was a novel use of `errors.As`, looked cool as heck. the error presenter code here is pretty clean btw, well done!